governance/default.go

```diff
@@ -521,18 +521,41 @@ func (g *Governance) ClearVotes(num uint64) {
 // parseVoteValue parse vote.Value from []uint8 to appropriate type
 func (g *Governance) ParseVoteValue(gVote *GovernanceVote) (*GovernanceVote, error) {
 	var val interface{}
-	k := GovernanceKeyMap[gVote.Key]
+	k, ok := GovernanceKeyMap[gVote.Key]
+	if !ok {
+		logger.Warn("Unknown key was given", "key", k)
+		return nil, ErrUnknownKey
+	}
 
-	// filter out if vote value is an interface list
-	if reflect.TypeOf(gVote.Value) == reflect.TypeOf([]interface{}{}) {
-		return nil, ErrValueTypeMismatch
+	// checking type of gVote.Value
+	switch k {
+	case params.AddValidator, params.RemoveValidator:
+		// gVote.Value type should be [][]interface{}(multiple node removal/addition) or []uint8(single node removal/addition)
+		if reflect.TypeOf(gVote.Value) != reflect.TypeOf([]interface{}{}) && reflect.TypeOf(gVote.Value) != reflect.TypeOf([]uint8{}) {
```

Comment:
```suggestion
		if reflect.TypeOf(gVote.Value) != reflect.TypeOf([][]interface{}{}) && reflect.TypeOf(gVote.Value) != reflect.TypeOf([]uint8{}) {
```

Need double `[]`?

Reply:
Or you need to check once more after casting []interface{}.

Reply:
Actually we only allow `[][]uint8`, don't we?

Reply:
Yes, we only allow [][]uint8. Current code doesn't check type of the inner array.. Thanks for the advice! I fixed the code to check the inner array type, too! reflected at https://github.com/klaytn/klaytn/pull/1110/commits/b1e90f018639923869c1fd9e7521b53032ecd69e